Automattic/wp-calypso
# A4A: Sites dashboard multiscreen size compatibility

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/115

## Proposed Changes

* This PR improves the compatibility with smaller resolutions as proposed in https://github.com/Automattic/automattic-for-agencies-dev/issues/115
* As a side effect, this PR also fixes some scrolling issues with all resolutions

## Testing Instructions

* **As a general guideline, ensure that everything appears visually appealing and provides a pleasant user experience.**

Small screens:
* Go to `http://agencies.localhost:3000/sites`
* Adjust your browser to use a mobile view (F12 -> Select a small resolution)
* In trunk you should see:
![image](https://github.com/Automattic/wp-calypso/assets/37049295/e9178dd6-fc49-436f-bc67-2fcd86927b55)


* In `update/a4a/115-sites-dashboard-multiscreen-size-compatibility` you should see :
![image](https://github.com/Automattic/wp-calypso/assets/37049295/f242dc22-66f9-4faa-b757-78a47b2fb9ad)

* When visiting other pages, the page's top margin should be fixed (e.g.  `http://agencies.localhost:3000/overview`)
* Trunk:
![image](https://github.com/Automattic/wp-calypso/assets/37049295/3b0f71ce-874e-47ae-974d-03e598a9b6bd)

* In `update/a4a/115-sites-dashboard-multiscreen-size-compatibility` you should see :
![image](https://github.com/Automattic/wp-calypso/assets/37049295/edc06058-cc74-4f6b-bcf0-6e84cc4992da)


Action buttons:
* Trunk:
![image](https://github.com/Automattic/wp-calypso/assets/37049295/55972687-b209-491b-8203-d3dad45dec48)

* In `update/a4a/115-sites-dashboard-multiscreen-size-compatibility` you should see :
![image](https://github.com/Automattic/wp-calypso/assets/37049295/75c2bb08-51b0-4021-ab2a-31341c188ee4)


## Pre-merge Checklist
